client/angular/controllers/AdminController.js

```diff
@@ -57,6 +57,8 @@
 
         $scope.setRetweetDisplayStatus = adminDashDataService.setRetweetDisplayStatus;
 
+        $scope.setApprovedTweetsOnlyStatus = adminDashDataService.setApprovedTweetsOnlyStatus;
```

Comment:
Not really related to this pull, but I'll allow it because it's weird that this isn't already in master